flow/flow_test.go

```diff
@@ -273,9 +285,8 @@ func validateAllParentChains(t *testing.T, table *Table) {
 
 func flowsFromPCAP(t *testing.T, filename string, linkType layers.LinkType, bpf *BPF) []*Flow {
 	table := NewTable(nil, nil, NewEnhancerPipeline(), "", TableOpts{})
-
+	table.SetSynCapture(true)
```

Comment:
as the flag is global to the agent, and will be set only by the user, set the config flag directly here for the purpose of the test, so place this code in your unit test (TestFlowSimpleSynFin)

config.GetConfig().Set("agent.capture_syn", true)
defer config.GetConfig().Set("agent.capture_syn", false)